vars/mtaBuild.groovy

```diff
@@ -28,6 +31,16 @@ def call(Map parameters = [:]) {
                                       parameters, parameterKeys,
                                       stepConfiguration, stepConfigurationKeys, stepDefaults)
 
+
+        MTA_JAR_FILE_VALIDATE: {
+            // same order like inside getMtaJar,
+            def mtaJarLocation = configuration?.mtaJarLocation ?: env?.MTA_JAR_LOCATION
+            def returnCodeLsMtaJar = sh script: "ls ${DEFAULT_MTA_JAR_NAME}", returnStatus:true
+            if(mtaJarLocation || ( !mtaJarLocation && returnCodeLsMtaJar != 0)) {
```

Comment:
I would directly fail here if ( !mtaJarLocation && returnCodeLsMtaJar != 0) instead of passing an empty mtaJarLocation to toolValidate. This would be better readable/understandable.
And logging which one is used in the end could help.

Reply:
It was my aim to refactor as close to the status before this pull request. I fail in the same way like if failed before. There is a speaking error message provided AFAIR in case we have no mtaJarLocation.

Reply:
> And logging which one is used in the end could help.

done, we have now a log entry in case mta.jar is used from current working dir.